.circleci/config.yml

```diff
@@ -26,7 +26,6 @@ workflows:
     jobs:
       - run-specs-with-postgres
       - run-specs-with-mysql
-      - lint-code
```

Comment:
Why? 🤔

Reply:
I thought it is out of the scope of this PR to fix all the rubocop warnings emitted. What do you think?